src/linker/Linker/Driver.cs

```diff
@@ -1188,6 +1223,8 @@ static void Usage ()
 			Console.WriteLine ("                              VERSION is an integer in the range 0-9999.");
 			Console.WriteLine ("  --warnaserror[+|-]        Report all warnings as errors");
 			Console.WriteLine ("  --warnaserror[+|-] WARN   Report specific warnings as errors");
+			Console.WriteLine ("  --collapse[+|-]           Collapse analysis warnings to one per assembly");
```

Comment:
To me `--collapse System.Runtime` option is anything but a warning setting. We should change the option to be more descriptive because if I understood the code correctly it does not collapse anything but replaces any warning with a generic warning. What about something like `--warnasone` or `--singlewarn` ?

Reply:
Changed to `--singlewarn`. Thanks!